modules/animation/src/animationcontroller.cpp

```diff
@@ -362,7 +362,7 @@ void AnimationController::tick() {
     // What to do when network cannot be evaluated in the speed that is given by deltaTime?
     // Initial solution: Don't care about that, and let it evaluate fully in the speed that it can
     // muster.
```

Comment:
Can this comment be resolved now?

Reply:
Not really. I think that it might need to be determined from the outside